packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -110,6 +117,19 @@ export const FeedContainer = ({
   } as CSSProperties;
   const cardContainerStyle = { ...getStyle(isList, spaciness) };
   const suggestionsProps = useSearchSuggestions({ origin: Origin.HomePage });
+  const isTracked = useRef(false);
```

Comment:
Why is this useRef and not useState?

Reply:
We don't need the re-render for when it is tracked using a state. We just want to make sure it is tracked